src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40;
+  const littleSale = 20;
+  const bigSale = 50;
+  let fullCost = days * rentalCost;
+
+  if (days % 3 === 0) {
+    fullCost -= littleSale;
+  } else if (days % 7 === 0) {
+    fullCost -= bigSale;
+  }
```

Comment:
it works only if `day` is 3,6,9 ... or 7,14,21... 😢

Reply:
at first i thought that discount would be added every 3 or 7 days ;)